src/e2e-app/main-aot.ts

```diff
@@ -0,0 +1,4 @@
+import {platformBrowserDynamic} from '@angular/platform-browser-dynamic';
+import {E2eAppModuleNgFactory} from './e2e-app-module.ngfactory';
+
+platformBrowserDynamic().bootstrapModuleFactory(E2eAppModuleNgFactory);
```

Comment:
Shouldn't this just be `platformBrowser` (without the `Dynamic`)?

(same for demo-app, now that I notice it)